rust/src/rfb/parser.rs

```diff
@@ -36,7 +36,7 @@ pub enum RFBGlobalState {
     TSVncResponse,
     TCSecurityResult,
     TSClientInit,
-    Message
+    Message,
```

Comment:
please split formatting changes from the test additions into separate commits

Reply:
Ok.Will address in next PR